### PR TITLE
Ins 1018 mussel app db

### PR DESCRIPTION
**Pull request by fwpushan**

### CODE REVIEW

File: api/api_sources/schema-files/watercraftRiskAssessment.schema.yaml [hunk 1]
```diff
@@ -15,31 +16,140 @@ schemas:
...
-        name: full_observation_form
-        comment: 'Stored JSON structure for full observation questionnaire'
-        definition: JSONB NULL
-      additionalInfo:
-        name: additional_info
-        comment: 'Stored JSON structure for additional information'
-        definition: JSONB NULL
+      # Boolean
+      passportHolder:
+        name: 'pass_port_holder_ind'
```
Comment (andrea-williams) [inline api/api_sources/schema-files/watercraftRiskAssessment.schema.yaml]: should be 'passport_holder_ind' - 'passport' is more intuitive than 'pass_port'

File: api/api_sources/sources/database/models/highRiskAssessment.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+import { Column, Entity, PrimaryGeneratedColumn} from 'typeorm';
+import { HighRiskAssessmentSchema } from '../database-schema';
+import {
+} from '../database-schema';
+
+import { ModelProperty, PropertyType, ModelDescription } from '../../libs/core-model';
+import { IntTransformer } from '../../libs/transformer';
+import {
+} from '../models';
```
Comment (andrea-williams) [inline api/api_sources/sources/database/models/highRiskAssessment.ts]: refactor: delete empty imports ^

File: README.md [hunk 3]
```diff
@@ -135,17 +146,13 @@ This will perform hotloading of any changes made to the frontend code (i.e., you
...
 To quit the local client, type `<Ctrl> + C`.
 
-To run the server (backend) within a Docker container, execute
-`make local-debug`
-This will print additional logging statements to the console, which may be useful when debugging the backend.
-
 *View the App:*
 
-Go to http://localhost:4200/
+Go to http://localhost:3033/
```
Comment (andrea-williams) [inline README.md]: Shouldn't this still be port 4200?

File: api/api_sources/schema-files/watercraftRiskAssessment.schema.yaml (hunk 1 above)
Comment (fwpushan) [reply api/api_sources/schema-files/watercraftRiskAssessment.schema.yaml]: Resolved

File: api/api_sources/sources/database/models/highRiskAssessment.ts (hunk 2 above)
Comment (fwpushan) [reply api/api_sources/sources/database/models/highRiskAssessment.ts]: Resolved

File: api/api_sources/sources/database/models/watercraftRiskAssessment.ts [hunk 4]
```diff
@@ -91,49 +134,197 @@ export class WatercraftRiskAssessment extends Record implements WatercraftRiskAs
...
-		name: WatercraftRiskAssessmentSchema.columns.timestamp,
-		transformer: new DateTimeTransformer()
-	})
+	@Column({ name: WatercraftRiskAssessmentSchema.columns.timestamp, transformer: new DateTimeTransformer()})
 	@ModelProperty({type: PropertyType.string})
 	timestamp: string;
 
 	/**
-	 * @description Getter/Setter property for column {high_risk_assessment_form}
+	 * @description Getter/Setter property for column {pass_port_holder_ind}
```
Comment (andrea-williams) [inline api/api_sources/sources/database/models/watercraftRiskAssessment.ts]: passport_holder_ind

**Pull request merged**